app/src/Build/BuildDocsTask.php

```diff
@@ -358,8 +358,5 @@ private function getDoctum(): Doctum
     private function log(string $message)
     {
         $this->output->writeln($message);
-        if ($this->job) {
-            $this->job->addMessage(strip_tags($message));
-        }
```

Comment:
Why is this (and the corresponding constructor and property) being removed?

Reply:
Ahh, looks like you were correct to remove this initially - it's now outputting twice in the `messages` tab for the job:

```text
[2025-11-13 03:00:03][INFO] Making new version map
[2025-11-13 03:00:04][INFO] Updating bringyourownideas/silverstripe-maintenance
[2025-11-13 03:00:04][INFO] Running task 'SilverStripe\ApiDocs\Build\BuildDocsTask'
Making new version map
Updating bringyourownideas/silverstripe-maintenance
```

Task output must automatically be added to messages - so we shouldn't add it explicitly here after all.